src/assignment/components/AssignmentHeading.tsx

```diff
@@ -6,13 +9,11 @@ import {
 	ToolbarLeft,
 	ToolbarRight,
 } from '@viaa/avo2-components';
-import classnames from 'classnames';
+import classNames from 'classnames';
```

Comment:
prefer the migration towards clsx
its a smaller package and it is a drop in replacement for classnames
it should be already available in the avo-client repo
```import clsx from 'clsx';```